src/pages/Landing.tsx

```
@@ -0,0 +1,267 @@
+import { Link, useNavigate } from "react-router-dom";
+import { Button } from "@/components/ui/button";
+import { Briefcase, Users } from "lucide-react";
+import Layout from "@/components/layout/Layout";
+import VoteIcon from "@/components/\bicon/VoteIcon";
+import CommentIcon from "@/components/\bicon/CommnetIcon";
+
+const navigate = useNavigate();
```

ESLint

[error] React Hook "useNavigate" cannot be called at the top level. React Hooks must be called in a React function component or a custom React Hook function.

(react-hooks/rules-of-hooks)